modules/recommended-requirements-secured-cluster-services.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+// * cloud_service/acscs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-secured-cluster-services_{context}"]
+= Secured cluster services
+
+Secured cluster services contain the following components:
+
+* Sensor
+* Admission controller
+* Collector
+
+[NOTE]
+====
+Collector component is not included on this page. Required resource requirements are listed on the *default resource requirements* page.
```

Comment:
```suggestion
Collector component is not included on this page. Required resource requirements are listed on the default resource requirements page.
```

Reply:
I have moved this note under the `Collector` bullet point.